File: 2309/yeolyi.js
const fs = require('fs')
const arr = fs.readFileSync('/dev/stdin').toString().trim().split('\n').map(Number);
const total = arr.reduce((a, b) => a + b, 0);
const remaining = total - 100;

loop:
for (let i = 0; i < 8; i++) {
    for (let j = i + 1; j < 9; j++) {
        if (arr[i] + arr[j] == remaining) {
            console.log(arr
                .filter((_, idx) => idx != i && idx != j)
                .sort((a, b) => a - b)
                .join("\n")
            );
            break loop;
        }
    }
}

File: 2309/yeolyi.swift
let arr: [Int] = {
    var temp = [Int]()
    (0..<9).forEach { _ in 
        temp.append(Int(readLine()!)!)
    }
    return temp
}()

let remaining = arr.reduce(0, +) - 100

loop: for i in 0..<8 {
    for j in (i+1)..<9 where arr[i] + arr[j] == remaining {
        print(arr
            .sorted()
            .reduce("", {
                guard $1 != arr[i] && $1 != arr[j] else { return $0 }
                return "\($0)\n\($1)"
            })
        )
        break loop
    }
}
